core/src/main/scala/sttp/tapir/internal/package.scala

```diff
@@ -192,6 +191,22 @@ package object internal {
         Vector(b.bodyType)
       }.headOption
     }
+
+    def matchesContent(content: Either[String, MediaType]): Boolean = {
```

Comment:
I think this should be `def matchesContent(content: MediaType): Boolean`

The either is a detail external to this method